diesel_derives/tests/queryable_by_name.rs

```diff
@@ -18,11 +19,19 @@ fn named_struct_definition() {
     struct MyStruct {
         foo: i32,
         bar: i32,
+        r#type: i32,
     }
 
     let conn = &mut connection();
-    let data = sql_query("SELECT 1 AS foo, 2 AS bar").get_result(conn);
-    assert_eq!(Ok(MyStruct { foo: 1, bar: 2 }), data);
+    let data = sql_query("SELECT 1 AS foo, 2 AS bar, 3 AS `r#type`").get_result(conn);
```

Comment:
QueryableByName perhaps poses an interesting question. Should the query indeed contain `` `r#type` `` to match or should the SQL query contain `type` and map to the raw identifier `r#type`?

Reply:
I would expect `type` there.